rktlet/runtime/helpers.go

```diff
@@ -323,6 +323,13 @@ func generateAppAddCommand(req *runtimeApi.CreateContainerRequest, imageID strin
 			glog.Warningf("unexpected nil mount: %v, %+v", mnt, config)
 			continue
 		}
+
+		if err := createHostPathVolume(mnt); err != nil {
+			glog.Errorf("create volume HostPath %q for Pod %q failed: %v", mnt.HostPath, req.PodSandboxId, err)
+			return nil, err
+		}
+		glog.V(4).Infof("created volume HostPath %q for Pod %q", mnt.HostPath, req.PodSandboxId)
```

Comment:
shouldn't you add the log only if the volume dir was created? (move that instruction inside `createHostPathVolume()`)